Sources/TuistDependencies/Models/Graph.swift

```diff
@@ -0,0 +1,33 @@
+import Foundation
+import TuistCore
+
+struct Graph: Codable, Equatable {
+    let iOSDependencies: [String]
+    let tvOSDependencies: [String]
+    let macOSDependencies: [String]
+    let watchOSDependencies: [String]
```

Comment:
I think the structure of this graph has been specifically designed for Carthage. This requires further thinking to make sure that the structure works for the other package managers that we plan to support: CocoaPods and SPM. Would it be ok to leave this out of the PR and move the discussion to an issue?